<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editor_bar_Window</class>
 <widget class="QMainWindow" name="Editor_bar_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>518</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Editor Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="Page_control_frame_editor">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>470</y>
      <width>551</width>
      <height>44</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QPushButton" name="BackButton_editor">
       <property name="text">
        <string>TempBack</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Current_page_label_editor">
       <property name="contextMenuPolicy">
        <enum>Qt::ContextMenuPolicy::ActionsContextMenu</enum>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Plain</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="NextButton_editor">
       <property name="text">
        <string>TempNext</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Add_page_button_editor">
       <property name="baseSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>+</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Remove_page_button_editor">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_bar_button">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="load_bar_button">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="Ready_Widget_list">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>261</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="Bar_editor_QFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>281</width>
      <height>451</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <property name="verticalSpacing">
      <number>4</number>
     </property>
     <item row="0" column="0">
      <widget class="QFrame" name="Bar_editor_target_1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>260</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>260</width>
         <height>80</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QFrame" name="Bar_editor_target_2">
       <property name="minimumSize">
        <size>
         <width>260</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>260</width>
         <height>80</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QFrame" name="Bar_editor_target_3">
       <property name="minimumSize">
        <size>
         <width>260</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>260</width>
         <height>80</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QFrame" name="Bar_editor_target_4">
       <property name="minimumSize">
        <size>
         <width>260</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>260</width>
         <height>80</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QFrame" name="Bar_editor_target_5">
       <property name="minimumSize">
        <size>
         <width>260</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>260</width>
         <height>80</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
